ambry-api/src/main/java/com.github.ambry/rest/RestServiceErrorCode.java

```diff
@@ -130,6 +134,8 @@ public static RestServiceErrorCode getRestServiceErrorCode(RouterErrorCode route
         return Deleted;
       case BlobDoesNotExist:
         return NotFound;
+      case RangeNotSatisfiable:
+        return RangeNotSatisfiable;
       case AmbryUnavailable:
       case InsufficientCapacity:
       case OperationTimedOut:
```

Comment:
I'll make `ChannelClosed` map to an internal error. It should only occur after the getBlob callback has already been called, because it requires the user to close the channel it received.